modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
 
----
+*Requires Restart:* No
 
-=== cloud_storage_disable_tls
+*Nullable:* No
 
-Disable TLS for all S3 or GCS connections.
+*Visibility:* `tunable`
 
-*Type*: boolean
+*Type:* integer
 
-*Default*: false
+*Accepted values:* [`-17592186044416`, `17592186044415`]
 
-*Restart required*: yes
+*Default:* `500ms`
 
 ---
 
-=== cloud_storage_enabled
+=== audit_queue_max_buffer_size_per_shard
+
+Defines the maximum amount of memory in bytes used by the audit buffer in each shard. Once this size is reached, requests to log additional audit messages will return a non-retryable error.
 
-Enable object storage. Must be set to `true` to use xref:manage:tiered-storage.adoc[Tiered Storage] or Remote Read Replicas.
+*Requires Restart:* Yes
 
-*Type*: boolean
+*Nullable:* No
 
-*Default*: false
+*Visibility:* `tunable`
 
-*Restart required*: yes
+*Type:* integer
+
+*Default:* `1048576`
 
 ---
 
-=== cloud_storage_max_connections
+=== auto_create_topics_enabled
 
-Max number of simultaneous connections to S3 per shard. Includes connections used for both uploads and downloads.
+Allow automatic topic creation. To prevent runaway topic creation, this property is not supported for Redpanda Cloud BYOC and dedicated clusters. You should explicitly manage topic creation for these Redpanda Cloud clusters.
 
-*Units*: number of simultaneous connections
+*Type*: boolean.
 
-*Default*: 20
+*Requires Restart:* No
 
-*Restart required*: yes
+*Nullable:* No
 
----
+*Visibility:* `user`
 
-=== cloud_storage_region
+*Type:* boolean
 
-AWS or GCP region that houses the bucket used for storage.
+*Default:* `false`
 
-*Type*: string
+---
 
-*Default*: null
+=== cluster_id
 
-*Restart required*: yes
+Cluster identifier.
 
----
+*Type*: string.
 
-=== cloud_storage_secret_key
+*Requires Restart:* No
 
-AWS or GCP secret key.
+*Nullable:* Yes
 
-*Type*: string
+*Visibility:* `None`
 
-*Default*: null
+*Type:* string
 
-*Restart required*: yes
+*Default:* `null`
 
 ---
 
-=== cloud_storage_trust_file
+=== compacted_log_segment_size
 
-Path to certificate that should be used to validate server certificate during TLS handshake.
+How large in bytes should each compacted log segment be (default 256MiB).
 
-*Type*: string
+*Requires Restart:* No
 
-*Default*: null
+*Nullable:* No
 
-*Restart required*: yes
+*Visibility:* `tunable`
 
----
+*Type:* integer
 
-== Cluster management properties
+*Accepted values:* [`0`, `18446744073709551615`]
 
-=== cluster_id
+*Default:* `268435456`
 
-Cluster identifier.
+---
 
-*Type*: string
+=== compaction_ctrl_backlog_size
 
-*Default*: null
+Target backlog size for compaction controller. if not set compaction target compaction backlog would be equal to .
```

Comment:
```suggestion
Target backlog size for compaction controller. If not set, compaction target backlog equals .
```
ENG: This is missing info; e.g., If not set, compaction target backlog equals _what_?